apps/coscrad-frontend/src/emotion-components/theme.ts

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+import { createTheme } from '@mui/material';
+import { primaryColor, secondaryColor } from './colors';
+
+export const theme = createTheme({
+    palette: {
+        primary: {
+            main: primaryColor[100],
```

Comment:
Can I just point out how weird it is that they are using numeric keys here? It looks like an array that has one single element indexed at `100`. I really prefer to use string keys for non-array objects (Arrays are implemented as objects with numeric keys in JS).